e2e/tests/transfer/authz_test.go

```diff
@@ -53,6 +53,7 @@ func (suite *AuthzTransferTestSuite) TestAuthz_MsgTransfer_Succeeds() {
 	ctx := context.TODO()
 
 	testName := t.Name()
+	t.Parallel()
```

Comment:
Is there a rule of thumb when to add this?

Reply:
Ideally we can just add this any time there are tests that do not make chain-wide modifications such as  param changes / chain halts / config modifications etc.

In some cases it works out the box (like with authz and forwarding E2Es) and some will take some re-working or tweaking (looking at ICA )

Reply:
Will make sure to add this to the updated README for E2Es once this block of work is done!